### PR TITLE
FIXED: data charts

### PR DESCRIPTION
**Pull request by mactanxin**
ticket: 20302

### CODE REVIEW

File: src/dispatcher/src/frontend/static/js/app.js [hunk 1]
```diff
@@ -3,12 +3,30 @@
...
+  'ngAnimate',
+  'pascalprecht.translate'
 ]);
+
+DebuggerApp.config(['$translateProvider', function ($translateProvider) {
+  $locationProvider.html5Mode(true);
+  $translateProvider.translations('en', {
+    'TITLE': 'Hello',
+    'FOO': 'This is a paragraph'
+  });
```
Comment (jceel) [inline src/dispatcher/src/frontend/static/js/app.js]: Is that related to charts in any way?

Comment (mactanxin) [inline src/dispatcher/src/frontend/static/js/app.js]: nope, that's for translation

Comment (jceel) [inline src/dispatcher/src/frontend/static/js/app.js]: OK, then please submit it as another pull request

**Comment by mactanxin** [comment]
updated
